.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,4 @@
-blank_issues_enabled: true
+blank_issues_enabled: false
```

Comment:
It's a very small option in the interface, maybe we should keep that so a very specific issue with a lot of work (for performance for example) do not get shoehorned into our bug template.

Reply:
Tbh I wanted to set this to `enabled` but didn't realize this this was already the case 😅
Thanks for pointing it out.